docs/helm/helm_create.md

```diff
@@ -30,6 +30,13 @@ do not exist, Helm will attempt to create them as it goes. If the given
 destination exists and there are files in that directory, conflicting files
 will be overwritten, but other files will be left alone.
 
+Its aim is to show a basic chart with resources that can be installed to demonstrate
```

Comment:
This line isn't necessary. The chart is described [here](https://github.com/helm/helm/blob/master/cmd/helm/create.go#L33). This is an adequate description already.

Reply:
get it.